src/systems/filecoin_mining/storage_mining/storage_miner_actor_code.go

```diff
@@ -99,31 +99,62 @@ func (a *StorageMinerActorCode_I) _slashDealsFromFaultReport(rt Runtime, sectorN
 
 	Release(rt, h, st)
 
-	// TODO: Send(StorageMarketActor, ProcessDealSlash)
+	processDealSlashParam := make([]actor.MethodParam, 2)
+	rt.SendCatchingErrors(&msg.InvocInput_I{
+		To_:     addr.StorageMarketActorAddr,
+		Method_: market.MethodProcessDealSlash,
+		Params_: processDealSlashParam,
+	})
```

Comment:
Blocking: this should not be catching errors. there's no good reason for the call to fail, and if it does (out of gas?) the whole invocation chain should fail too. Defensive programming like this will become really hard to reason about.

If there's some really good justification for why a failure here must be recovered while the rest of this succeeds, it needs a long comment explaining it.

Reply:
+1. Also, with #644 , we now have an extra return value from `SendCatchingErrors` (namely, the error exit code). Calls to `SendCatchingErrors` should never ignore that exit code, except in very special cases such as CronActor.

Reply:
(Separately, @anorth currently the plan is for out-of-gas to be a special-cased unconditional failure, which cannot be caught even by `SendCatchingErrors`, except in the case of Cron.)